clients/trieve-shopify-extension/extensions/global-search/blocks/inline_component.liquid

```diff
@@ -67,6 +68,7 @@
       getCartQuantity: (trackingId) => {
         return checkCartQuantity(parseInt(trackingId));
       },
+      currency: currency,
```

Comment:
I had this formatted to where the fields were in the same order for inline and global. This should also go right above the `{% comment %}` line the same it is in the `global_component.liquid`.